tardis/tardis_portal/api.py

```diff
@@ -872,3 +884,20 @@ def hydrate(self, bundle):
             bundle.data['file_object'].close()
             del(bundle.data['file_object'])
         return bundle
+
+
+class ObjectAclResource(MyTardisModelResource):
+    content_object = GenericForeignKeyField({
+        Experiment: ExperimentResource,
+        # ...
+    }, 'content_object')
+
+    class Meta:
+        authentication = default_authentication
+        authorization = ACLAuthorization()
+        queryset = ObjectACL.objects.all()
+        filtering = {
+            'object_id': ('exact', ),
```

Comment:
Going with the simplest option for now - removing "object_id", rather than adding "content_type", because there's still a lot more work to do to get all the required changes merged from the "mydata" branch's api.py